doc/Programming-guide/Key-platform-features/Configuration-management/index.md

```diff
@@ -437,208 +514,477 @@ The base schema is obtained by transforming the configuration schema, as follows
 
-The user-specific configuration management allows updating configuration data for the specific user under the application. The user-specific configuration management implements the same approach as in [Group-specific configuration]() management, based on the override schema and override algorithm.
+The user-specific configuration management allows updating configuration data for the specific user under the application. The user-specific
+configuration management implements the same approach as in [Group-specific configuration](#group-specific-configuration-management) management,
+based on the override schema and override algorithm.
 
-**NOTE:**
-Since each endpoint belongs to only one user at a time, the override algorithm does not support data set merges as in the group-specific configuration management.  
+>**NOTE:**
+> Since each endpoint belongs to only one user at a time, the override algorithm does not support data set merges as in the group-specific
+configuration management. <br/>
 The override algorithm applies user-specific overrides only after all group-specific overrides have been applied.
 
-## Endpoint data synchronization
+# Endpoint data synchronization
 
-After calculating the up-to-date configuration for the endpoint, the Operations server constructs a delta update based on the knowledge of the prior configuration of that endpoint. In certain cases, the delta update can contain a complete endpoint configuration.
+After calculating the up-to-date configuration for the endpoint, the Operations service constructs a delta update based on the knowledge of the
+prior configuration of that endpoint. In certain cases, the delta update can contain a complete endpoint configuration.
 
-Having received the delta update, the endpoint merges it with the existing configuration data, notifies the client application about the update, and persists the resulting configuration. The data storage location is set as an abstraction in the endpoint, therefore the concrete location for persisting the data is defined in the client implementation.
+Having received the delta update, the endpoint merges it with the existing configuration data, notifies the client application about the update, and
+persists the resulting configuration. The data storage location is set as an abstraction in the endpoint, therefore the concrete location for
+persisting the data is defined in the client implementation.
 
-By delivering configuration deltas instead of full configuration sets each time the endpoint needs to be reconfigured, the Kaa server significantly reduces a traffic load in the available data channels.
+By delivering configuration deltas instead of full configuration sets each time the endpoint needs to be reconfigured, the Kaa server significantly
+reduces a traffic load in the available data channels.
 
 Data consistency is ensured by the hash comparison between the endpoint and the server.
 
-### Protocol schema
+## Protocol schema
```

Comment:
Completely remove this and everything related to delta calculation from the doc. This feature is not supported.